hazelcast-sql/src/test/java/com/hazelcast/sql/misc/SqlNoDeserializationTest.java

```diff
@@ -166,43 +148,7 @@ public void testClient() {
                 clientService.invokeOnConnection(connection, executeRequest)
         );
 
-        if (executeResponse.error != null) {
```

Comment:
Why are we removing all this logic?

Reply:
Because it fails in above statements.

Reply:
But should it fail there?

Reply:
In this case, I don't know.
WDYT @viliam-durina @gierlachg ?

Reply:
Surely, it wasn't failing with IMDG engine, was it?

Reply:
Same story : it fails-fast on `execute` and returns exception : 

```
com.hazelcast.jet.JetException: Exception in ProcessorTasklet{0699-c15d-5480-0001/Project(IMap[partitioned.map])#0}: com.hazelcast.nio.serialization.HazelcastSerializationException: java.io.IOException: KEY FAILURE
```
I think this test was implemented taking into account the IMDG engine specifics. Jet produces same error, but earlier.
We can create follow up issue to dig deeper and potentially equalize IMDG and Jet behavior.

Reply:
I think that entire idea was to verify that an exception isn't produced early.

Reply:
https://github.com/hazelcast/hazelcast/issues/19273